src/VisualStudio/Core/Def/StackTraceExplorer/StackFrameViewModel.cs

```diff
@@ -167,39 +176,66 @@ public async Task NavigateToFileAsync(CancellationToken cancellationToken)
 
         protected override IEnumerable<Inline> CreateInlines()
         {
-            yield return MakeClassifiedRun(ClassificationTypeNames.Text, _frame.GetTextBeforeType());
+            var methodDeclaration = _frame.Root.MethodDeclaration;
+            var tree = _frame.Tree;
+            var className = methodDeclaration.MemberAccessExpression.Left;
+            var leadingTrivia = className.GetLeadingTrivia();
+            yield return MakeClassifiedRun(ClassificationTypeNames.Text, leadingTrivia.CreateString());
```

Comment:
don't love IEnuemrables.  it makes it unclear to me on the consumption side that we're not accidentallly running all those code multiple times.

Reply:
🤷‍♂️ ` yield return` is easier to find than `builder.Add`. Agree it does make consumption ambiguous. Other than intrinsic knowledge of how property binding works... which guarantees this is only read once. 

I'll think on it more, may just switch to `ImmutableArray` + builder.